pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -356,6 +361,14 @@ public String getControllerVipProtocol() {
         .orElse(CommonConstants.HTTP_PROTOCOL);
   }
 
+  public String getControllerBrokerProtocol() {
+    return Optional.ofNullable(getProperty(CONTROLLER_BROKER_PROTOCOL))
```

Comment:
Validate the protocol?

Reply:
validation is right below

Reply:
Throw exception when the protocol configured is not supported?

Reply:
done